e2e/tests/nightly/admin/workspace-admin-ui.spec.ts

```diff
@@ -0,0 +1,110 @@
+    await workspaceAdminPage.waitForLoad();
+    await workspaceAdminPage.getWorkspaceNamespaceInput().type(workspaceNamespace);
+    await workspaceAdminPage.clickLoadWorkspaceButton();
+    await workspaceAdminPage.waitForLoad();
+    expect(await workspaceAdminPage.getWorkspaceHeader()).toEqual('Workspace');
+    const headings3 = await workspaceAdminPage.getAllHeadings3();
+    expect(headings3).toEqual(
+      expect.arrayContaining([
+        WorkspaceHeadings.BasicInformation,
+        WorkspaceHeadings.Collaborators,
+        WorkspaceHeadings.CohortBuilder,
+        WorkspaceHeadings.CloudStorageObjects,
+        WorkspaceHeadings.ResearchPurpose
+      ])
+    );
+    const headings2 = await workspaceAdminPage.getAllHeadings2();
+    expect(headings2).toEqual(
+      expect.arrayContaining([WorkspaceHeadings.CloudStorageTraffic, WorkspaceHeadings.Runtimes])
+    );
+    const cloudStorageCols = await workspaceAdminPage.getcloudStorageColNames();
+    expect(cloudStorageCols).toEqual(
+      expect.arrayContaining([CloudStorageHeader.Location, CloudStorageHeader.Filename, CloudStorageHeader.FileSize])
+    );
+    expect(await workspaceAdminPage.getRuntimeDeleteButton().exists()).toBeFalsy();
+    expect(await workspaceAdminPage.getNoActiveRuntimeText()).toEqual(noActiveRuntimeText);
+  });
+
+  test('Verify that admin is able to preview the Notebook', async () => {
+    const workspaceAdminPage = new WorkspaceAdminPage(page);
+    await workspaceAdminPage.waitForLoad();
+    await workspaceAdminPage.getWorkspaceNamespaceInput().type(workspaceNamespace);
+    workspaceAdminPage.clickLoadWorkspaceButton();
+    await workspaceAdminPage.waitForLoad();
+    //verify that the Notebook Preview button is disabled
+    expect(await workspaceAdminPage.getNotebookPreviewButton().isCursorNotAllowed()).toBe(true);
+    const accessReasonText = 'for auditing purposes';
+    await workspaceAdminPage.getAccessReasonInput().type(accessReasonText);
+    //verify that the Notebook Preview button is now enabled
+    expect(await workspaceAdminPage.getNotebookPreviewButton().isCursorNotAllowed()).toBe(false);
+    await workspaceAdminPage.clickNotebookPreviewButton();
+    const adminNotebookPreviewPage = new AdminNotebookPreviewPage(page);
+    await adminNotebookPreviewPage.waitForLoad();
+    // verify that the notebook preview page loaded
+    const previewCode = await adminNotebookPreviewPage.getFormattedCode();
+    expect(previewCode.some((item) => item.includes('import pandas'))).toBe(true);
+    expect(previewCode.some((item) => item.includes('import os'))).toBe(true);
+    expect(await adminNotebookPreviewPage.getNamespaceText()).toEqual(workspaceNamespace);
+    // click on the namespace link to navigate back to the workpsace admin page
+    await adminNotebookPreviewPage.clickNamespaceLink();
+    await workspaceAdminPage.waitForLoad();
+  });
+
+  test('Verify that admin is able to delete runtime', async () => {
```

Comment:
This test coverage is no longer testing `workspace-admin-ui` page. It should be removed or belongs in a different test suite.